cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -1650,7 +1650,7 @@ public int getTerminalWidth() {
     @Override
     public int getTerminalHeight() {
         if( !INTERACT ){
-            return 0;
+            return 24;
```

Comment:
Perhaps add a comment saying that 24 has no special meaning except that it is a value greater than 0.